extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmAnnotations.java

```diff
@@ -27,20 +12,20 @@ private HibernateOrmAnnotations() {
     }
 
     public static final List<DotName> PACKAGE_ANNOTATIONS = Collections.unmodifiableList(Arrays.asList(
-            DotName.createSimple(AnyMetaDef.class.getName()),
-            DotName.createSimple(AnyMetaDefs.class.getName()),
-            DotName.createSimple(FetchProfile.class.getName()),
-            DotName.createSimple(FetchProfile.FetchOverride.class.getName()),
-            DotName.createSimple(FetchProfiles.class.getName()),
-            DotName.createSimple(FilterDef.class.getName()),
-            DotName.createSimple(FilterDefs.class.getName()),
-            DotName.createSimple(GenericGenerator.class.getName()),
-            DotName.createSimple(GenericGenerators.class.getName()),
-            DotName.createSimple(ListIndexBase.class.getName()),
-            DotName.createSimple(NamedNativeQueries.class.getName()),
-            DotName.createSimple(NamedNativeQuery.class.getName()),
-            DotName.createSimple(NamedQueries.class.getName()),
-            DotName.createSimple(NamedQuery.class.getName()),
-            DotName.createSimple(TypeDef.class.getName()),
-            DotName.createSimple(TypeDefs.class.getName())));
+            DotName.createSimple("org.hibernate.annotations.AnyMetaDef"),
+            DotName.createSimple("org.hibernate.annotations.AnyMetaDefs"),
+            DotName.createSimple("org.hibernate.annotations.FetchProfile"),
+            DotName.createSimple("org.hibernate.annotations.FetchProfile$FetchOverride"),
+            DotName.createSimple("org.hibernate.annotations.FetchProfiles"),
+            DotName.createSimple("org.hibernate.annotations.FilterDef"),
+            DotName.createSimple("org.hibernate.annotations.FilterDefs"),
+            DotName.createSimple("org.hibernate.annotations.GenericGenerator"),
+            DotName.createSimple("org.hibernate.annotations.GenericGenerators"),
+            DotName.createSimple("org.hibernate.annotations.ListIndexBase"),
+            DotName.createSimple("org.hibernate.annotations.NamedNativeQueries"),
+            DotName.createSimple("org.hibernate.annotations.NamedNativeQuery"),
+            DotName.createSimple("org.hibernate.annotations.NamedQueries"),
+            DotName.createSimple("org.hibernate.annotations.NamedQuery"),
+            DotName.createSimple("org.hibernate.annotations.TypeDef"),
+            DotName.createSimple("org.hibernate.annotations.TypeDefs")));
```

Comment:
No tests here, either?

Reply:
It's tested by `HibernateOrmAnnotationsTest.testNoMissingPackageLevelAnnotation`